dist/templates/ovnkube-node.yaml.j2

```diff
@@ -162,8 +162,10 @@ spec:
         securityContext:
           runAsUser: 0
           capabilities:
-            add: ["NET_ADMIN", "SYS_ADMIN", "SYS_PTRACE"]
-          {% if kind is defined and kind -%}
+            add:
+            - NET_ADMIN
+            {% if ovn_unprivileged_mode=="no" -%}
+            - SYS_ADMIN
           privileged: true
           {% endif %}
```

Comment:
I dislike having a template `% if` rule that starts inside the `capabilities` param and ends after the following param.

But we shouldn't need any capabilities if we're `privileged` (or rather, we should automatically get all of them) so I think this can just be

    {% if ovn_unprivileged_mode=="no" %}
    privileged: true
    {% else %}
    capabilities:
      add:
        - NET_ADMIN
    {% endif %}

or something?

Reply:
I didn't like it either... let me retest with a proposed change.

Reply:
@danwinship verified and updated PR